### PR TITLE
Turned on SwiftFormat for the entire project

### LINKED ISSUE #6583: Progressively run files through SwiftFormat by temporarily uncommenting "--enable indent" in ".swiftformat"
The goal with "SwiftFormat" is to eventually have "--enable indent" uncommented in ".swiftformat"

We don't want a huge and unnecessary commit that modifies the entire codebase, so this GitHub issue is to do this:

1. Uncomment "--enable indent" in ".swiftformat"
2. "pods/SwiftFormat/CommandLineTool/swiftformat --config .swiftformat <a small subset of codebase>"
3. Undo (1), commit (2)
4. PR
5. Repeat 1-4 between other work, gradually

Feel free to suggest a better way.

One alternative that is cleaner:

1. Make a huge PR that adds these to every file:

"""
//swiftformat:disable --indent
<code...>
//swiftformat:enable --indent
"""
2. Uncomment "--enable indent" in ".swiftformat"
3. PR
4. Fix 1-2 files at a time
5. PR
6. Repeat 4-6 between other work, gradually or while touching files in other PRs

### PR DESCRIPTION
**Pull request by eviltofu**
Closes #6583 if accepted.

See https://github.com/AlphaWallet/alpha-wallet-ios/issues/6583#issuecomment-1532685415 onwards

### CODE REVIEW

File: AlphaWallet/Accounts/ViewModels/AccountsViewModel.swift [hunk 1]
```diff
@@ -64,7 +64,6 @@ final class AccountsViewModel {
          walletBalanceService: WalletBalanceService,
          blockiesGenerator: BlockiesGenerator,
          domainResolutionService: DomainResolutionServiceType) {
-
```
Comment (oa-s) [inline AlphaWallet/Accounts/ViewModels/AccountsViewModel.swift:67]: could u please not remove first space in init method, a bit easier to look at code with it

**Comment by eviltofu** [comment]
@hboon How should we proceed? I for doing everything at once.

**Comment by hboon** [comment]
A subset definitely. Otherwise merging is a nightmare. The smaller pods that don't get updated often and don't have live PRs can go first. If some files always seem to be active, we can leave it for a while

**Comment by hboon** [comment]
@eviltofu nah, just make a smaller set. Pick the pods that don't get modified much. We can get those in quickly. We don't make PRs that modify 1000 files unless we can avoid it. It's way easier to merge and you don't have to coordinate (much).

**Comment by hboon** [comment]
@eviltofu Let's close this for now.

**Pull request closed without merging**